tests/test_inner.py

```diff
@@ -68,7 +69,18 @@ def test_build_results_decoder():
         assert getattr(results, attr) == getattr(expected_results, attr)
 
 
+class MockDocker(object):
```

Comment:
Why would we want to create a new copy of MockDocker and not reuse one in `test_docker_api`? `inner.py` might not always use docker btw

Reply:
well they both can share that class, but we don't need data in that docker_api, but we need it in inner

Reply:
It would be easier to unify those, so that `MockDocker` class from `tests/plugins/test_docker_api.py` would not return empty history always

Reply:
it might as well break other test, and data are really only needed for inner